File: TestPayrollSystem/Mothers/PayPeriodMother.vb
Option Strict On

Imports AccuPay.Core.Entities

Public Class PayPeriodMother

    Public Shared Function StartDateOnly(payFromDate As Date) As PayPeriod

        Dim payPeriod As New PayPeriod() With
        {
            .PayFromDate = payFromDate
        }

        Return payPeriod

    End Function

End Class

File: AccuPay.Core.UnitTests/Mothers/PayPeriodMother.cs
using AccuPay.Core.Entities;
using System;

namespace AccuPay.Core.UnitTests
{
    public static class PayPeriodMother
    {
        public static PayPeriod StartDateOnly(DateTime payFromDate)
        {
            PayPeriod payPeriod = new PayPeriod()
            {
                PayFromDate = payFromDate
            };

            return payPeriod;
        }
    }
}
